staging/src/k8s.io/kubectl/pkg/cmd/create/create_pdb_test.go

```diff
@@ -17,69 +17,203 @@ limitations under the License.
 package create
 
 import (
-	"bytes"
-	"io/ioutil"
-	"net/http"
 	"testing"
 
-	"k8s.io/apimachinery/pkg/runtime/schema"
-	"k8s.io/cli-runtime/pkg/genericclioptions"
-	restclient "k8s.io/client-go/rest"
-	"k8s.io/client-go/rest/fake"
-	cmdtesting "k8s.io/kubectl/pkg/cmd/testing"
-	"k8s.io/kubectl/pkg/scheme"
+	policyv1beta1 "k8s.io/api/policy/v1beta1"
+	apiequality "k8s.io/apimachinery/pkg/api/equality"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
-func TestCreatePdb(t *testing.T) {
-	pdbName := "my-pdb"
-	tf := cmdtesting.NewTestFactory().WithNamespace("test")
-	defer tf.Cleanup()
-
-	ns := scheme.Codecs.WithoutConversion()
-
-	tf.Client = &fake.RESTClient{
-		GroupVersion:         schema.GroupVersion{Group: "policy", Version: "v1beta1"},
-		NegotiatedSerializer: ns,
-		Client: fake.CreateHTTPClient(func(req *http.Request) (*http.Response, error) {
-			return &http.Response{
-				StatusCode: http.StatusOK,
-				Body:       ioutil.NopCloser(&bytes.Buffer{}),
-			}, nil
-		}),
-	}
-	tf.ClientConfigVal = &restclient.Config{}
+func TestCreatePdbValidation(t *testing.T) {
+	selectorOpts := []string{"app=nginx"}
+	podAmountOpts := []string{"3", "50%"}
```

Comment:
maybe here, instead of using an array of strings, using a pure string can be better.

PodDisruptionBudgetOpts expects a string anyway :)

Reply:
Done!